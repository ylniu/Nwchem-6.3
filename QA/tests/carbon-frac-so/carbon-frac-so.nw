echo

title "carbon-frac-so"

start carbon-frac-so

memory 128 mb

geometry units angstrom nocenter
symmetry c1
  C   0.0 0.0 0.0
end

basis
  * library 6-31G
end

# For no averaging
#set dft:avg_fon .false.
dft
  direct
  xc pbe0
  fon partial 6 electrons 1.8 filled 4
end

relativistic
 zora
end
task sodft energy


# For debugging
#set dft:debugfon .true.
