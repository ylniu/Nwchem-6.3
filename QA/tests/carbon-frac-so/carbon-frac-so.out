 argument  1 = carbon-frac-so.nw



============================== echo of input deck ==============================
echo

title "carbon-frac-so"

start carbon-frac-so

memory 128 mb

geometry units angstrom nocenter
symmetry c1
  C   0.0 0.0 0.0
end

basis
  * library 6-31G
end

# For no averaging
#set dft:avg_fon .false.
dft
  direct
  xc pbe0
  fon partial 6 electrons 1.8 filled 4
end

relativistic
 zora
end
task sodft energy


# For debugging
#set dft:debugfon .true.
================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 6.1.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2012
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = orion
    program         = ../../../bin/LINUX64/nwchem
    date            = Thu Jan  3 14:57:08 2013

    compiled        = Thu_Jan_03_14:26:29_2013
    source          = /home/niri/nwchem/nwchem-dev
    nwchem branch   = Development
    nwchem revision = 23287
    ga revision     = 10143
    input           = carbon-frac-so.nw
    prefix          = carbon-frac-so.
    data base       = ./carbon-frac-so.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =    4194305 doubles =     32.0 Mbytes
    stack    =    4194305 doubles =     32.0 Mbytes
    global   =    8388608 doubles =     64.0 Mbytes (distinct from heap & stack)
    total    =   16777218 doubles =    128.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                  carbon-frac-so
                                  --------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 C                    6.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      C                 12.000000


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 C                     0.00000000     0.00000000     0.00000000

  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem-dev/src/basis/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           6-31G                    on all atoms 



                                 NWChem DFT Module
                                 -----------------


                                  carbon-frac-so


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  C (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.04752490E+03  0.001835
  1 S  4.57369510E+02  0.014037
  1 S  1.03948690E+02  0.068843
  1 S  2.92101550E+01  0.232184
  1 S  9.28666300E+00  0.467941
  1 S  3.16392700E+00  0.362312

  2 S  7.86827240E+00 -0.119332
  2 S  1.88128850E+00 -0.160854
  2 S  5.44249300E-01  1.143456

  3 P  7.86827240E+00  0.068999
  3 P  1.88128850E+00  0.316424
  3 P  5.44249300E-01  0.744308

  4 S  1.68714400E-01  1.000000

  5 P  1.68714400E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 C                           6-31G                   5        9   3s2p


  WARNING: CD fitting not compatible with spinorbit
  WARNING: disabling CD fitting
  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     1
          No. of electrons :     6
           Alpha electrons :     3
            Beta electrons :     3
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          This is a Direct SCF calculation.
          AO basis - number of functions:     9
                     number of shells:     5
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                         PBE0 Method XC Functional
                     Hartree-Fock (Exact) Exchange  0.250          
          PerdewBurkeErnzerhof Exchange Functional  0.750          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          C                   0.70       49           4.0       434
          Grid pruning is: on 
          Number of quadrature shells:    49
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-10

 Performing spin-orbit DFT (SO-DFT) calculations
 -----------------------------------------------

          Performing ZORA calculations
          ----------------------------


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -37.66025415
     tr(P*S):    0.6000000E+01

 Read atomic ZORA corrections from ./carbon-frac-so.zora_so

 dft_zora_read_so: failed to open ./carbon-frac-so.zora_so
       Generating atomic ZORA corrections
       ----------------------------------


 Grid_pts file          = ./carbon-frac-so.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      3        Max. recs in file   =     37328


 Wrote atomic ZORA corrections to ./carbon-frac-so.zora_so


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -37.66025415

      Non-variational initial energy
      ------------------------------

 Total energy =     -37.298070
 1-e energy   =     -50.455020
 2-e energy   =      13.156950
 HOMO         =      -0.057689
 LUMO         =      -0.057689

  frac. electrons    5.80000000000000       vs                     6
     tr(P*S):    0.5800000E+01
   Time prior to 1st pass:      0.6
     tr(P*S):    0.5800000E+01

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):        4.16             4156503
          Stack Space remaining (MW):        4.19             4193242

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -37.6250568340 -3.76D+01  1.79D-02  2.48D-02     0.6
                                                     1.79D-02  2.48D-02
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     2    -37.6421700411 -1.71D-02  4.24D-03  5.27D-03     0.6
                                                     4.24D-03  5.27D-03
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     3    -37.6451921956 -3.02D-03  1.45D-03  6.24D-04     0.7
                                                     1.45D-03  6.24D-04
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     4    -37.6456146323 -4.22D-04  4.74D-04  7.63D-05     0.7
                                                     4.74D-04  7.63D-05
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     5    -37.6456749220 -6.03D-05  1.72D-04  1.05D-05     0.7
                                                     1.72D-04  1.05D-05
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     6    -37.6456843315 -9.41D-06  6.51D-05  1.54D-06     0.8
                                                     6.51D-05  1.54D-06
  Singularity in Pulay matrix. Error and Fock matrices removed. 
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     7    -37.6456858369 -1.51D-06  2.55D-05  2.33D-07     0.8
                                                     2.55D-05  2.33D-07
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     8    -37.6456860819 -2.45D-07  1.02D-05  3.64D-08     0.8
                                                     1.02D-05  3.64D-08
  Singularity in Pulay matrix. Error and Fock matrices removed. 
     tr(P*S):    0.5800000E+01
 d= 0,ls=0.5,diis     9    -37.6456861223 -4.03D-08  4.10D-06  5.82D-09     0.9
                                                     4.10D-06  5.82D-09


         Total DFT energy =    -37.645686128951
      One electron energy =    -49.839700218360
           Coulomb energy =     17.152774908753
    Exchange-Corr. energy =     -4.958760819344
 Nuclear repulsion energy =      0.000000000000

       Scaling correction =      0.009027622823

 Numeric. integr. density =      5.799999590183

     Total iterative time =      0.3s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    2  Occ=1.000000D+00  E=-1.049214D+01
              MO Center=  1.8D-22, -1.3D-24,  2.2D-23, r^2= 1.1D-10
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    10    -0.995500   0.000000  1 C  s          

 Vector    3  Occ=1.000000D+00  E=-6.466583D-01
              MO Center=  7.7D-20, -3.0D-20, -4.1D-20, r^2= 2.3D-07
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    15    -0.566236   0.000000  1 C  s                 11    -0.508125   0.000000  1 C  s          
    10     0.232249   0.000000  1 C  s          

 Vector    4  Occ=1.000000D+00  E=-6.466583D-01
              MO Center= -1.0D-13, -1.0D-13, -1.0D-13, r^2= 8.0D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
     6    -0.564434   0.045145  1 C  s                  2    -0.506507   0.040512  1 C  s          
     1     0.231510  -0.018517  1 C  s          

 Vector    5  Occ=3.000000D-01  E=-2.398739D-01
              MO Center=  5.9D-16,  6.5D-28, -2.7D-16, r^2= 1.2D-02
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    13     0.367317  -0.029067  1 C  py                 5    -0.028463  -0.359692  1 C  pz         
    12    -0.028463  -0.359692  1 C  px                17     0.279827  -0.022144  1 C  py         
     9    -0.021684  -0.274024  1 C  pz                16    -0.021684  -0.274024  1 C  px         

 Vector    6  Occ=3.000000D-01  E=-2.398739D-01
              MO Center=  3.5D-14, -1.4D-26,  4.8D-16, r^2= 3.2D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
     4     0.000000  -0.368465  1 C  py                 3     0.360816   0.000000  1 C  px         
    14    -0.360816   0.000000  1 C  pz                 8     0.000000  -0.280702  1 C  py         
     7     0.274881   0.000000  1 C  px                18    -0.274881   0.000000  1 C  pz         

 Vector    7  Occ=3.000000D-01  E=-2.396231D-01
              MO Center=  5.4D-14, -5.8D-25, -2.0D-16, r^2= 5.0D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    14     0.451563   0.000000  1 C  pz                 3     0.451563   0.000000  1 C  px         
    18     0.344609   0.000000  1 C  pz                 7     0.344609   0.000000  1 C  px         

 Vector    8  Occ=3.000000D-01  E=-2.396231D-01
              MO Center=  2.0D-16, -3.6D-26,  2.8D-14, r^2= 2.5D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    12    -0.320780   0.317820  1 C  px                 5     0.320780  -0.317820  1 C  pz         
    16    -0.244802   0.242543  1 C  px                 9     0.244802  -0.242543  1 C  pz         

 Vector    9  Occ=3.000000D-01  E=-2.396159D-01
              MO Center= -3.0D-17,  2.1D-26,  2.6D-16, r^2= 3.1D-03
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    13    -0.512962  -0.069623  1 C  py                17    -0.391494  -0.053136  1 C  py         
     5     0.035549  -0.261913  1 C  pz                12     0.035549  -0.261913  1 C  px         
     9     0.027131  -0.199897  1 C  pz                16     0.027131  -0.199897  1 C  px         

 Vector   10  Occ=3.000000D-01  E=-2.396159D-01
              MO Center=  1.8D-14, -5.6D-25,  3.9D-17, r^2= 1.7D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
     4     0.000000   0.517665  1 C  py                 8     0.000000   0.395083  1 C  py         
     3     0.264315   0.000000  1 C  px                14    -0.264315   0.000000  1 C  pz         
     7     0.201730   0.000000  1 C  px                18    -0.201730   0.000000  1 C  pz         

 Vector   11  Occ=0.000000D+00  E= 9.795227D-01
              MO Center= -1.0D-14,  1.9D-25, -4.6D-13, r^2= 6.3D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    17    -0.042060  -0.646871  1 C  py                 9    -0.640045   0.041616  1 C  pz         
    16    -0.640045   0.041616  1 C  px                13     0.039054   0.600643  1 C  py         
     5     0.594311  -0.038642  1 C  pz                12     0.594311  -0.038642  1 C  px         

 Vector   12  Occ=0.000000D+00  E= 9.795227D-01
              MO Center= -4.4D-13,  2.8D-25,  9.7D-15, r^2= 6.3D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
     8     0.000000  -0.648237  1 C  py                18    -0.641397   0.000000  1 C  pz         
     7     0.641397   0.000000  1 C  px                 4     0.000000   0.601911  1 C  py         
    14     0.595566   0.000000  1 C  pz                 3    -0.595566   0.000000  1 C  px         

 Vector   13  Occ=0.000000D+00  E= 9.799460D-01
              MO Center= -1.4D-14, -6.0D-24, -6.8D-13, r^2= 9.4D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
     9     0.784695  -0.074607  1 C  pz                16    -0.784695   0.074607  1 C  px         
     5    -0.728963   0.069308  1 C  pz                12     0.728963  -0.069308  1 C  px         

 Vector   14  Occ=0.000000D+00  E= 9.799460D-01
              MO Center= -6.7D-13,  2.1D-23,  1.3D-14, r^2= 9.5D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    18     0.788234   0.000000  1 C  pz                 7     0.788234   0.000000  1 C  px         
    14    -0.732250   0.000000  1 C  pz                 3    -0.732250   0.000000  1 C  px         

 Vector   15  Occ=0.000000D+00  E= 9.799521D-01
              MO Center=  3.4D-14, -2.9D-24, -1.5D-13, r^2= 2.8D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
    17    -0.329819  -0.823635  1 C  py                13     0.306399   0.765151  1 C  py         
    16     0.416206  -0.166667  1 C  px                 9     0.416206  -0.166667  1 C  pz         
    12    -0.386656   0.154834  1 C  px                 5    -0.386656   0.154834  1 C  pz         
     8     0.000000  -0.188964  1 C  py                 4     0.000000   0.175546  1 C  py         

 Vector   16  Occ=0.000000D+00  E= 9.799521D-01
              MO Center= -2.6D-13,  2.4D-23, -5.2D-14, r^2= 3.2D-01
  Bfn.         Coefficient        Function           Bfn.         Coefficient        Function  
  ----     -------------------  ------------         ----     -------------------  ------------
     8     0.000000   0.887217  1 C  py                 4     0.000000  -0.824219  1 C  py         
     7     0.448336   0.000000  1 C  px                18    -0.448336   0.000000  1 C  pz         
     3    -0.416505   0.000000  1 C  px                14     0.416505   0.000000  1 C  pz         
    17    -0.070246  -0.175422  1 C  py                13     0.065258   0.162965  1 C  py         


   alpha - beta orbital overlaps 
   ----------------------------- 


   alpha      1      2      3      4      5      6      7      8      9
    beta      1      6      3      4      5      6      3      4      5
 overlap   1.045  0.287  0.037  0.037  0.037  0.321  0.028  0.028  0.028

     --------------------------
     Expectation value of S2:  
     --------------------------
      <S2> =      0.7300 (Exact =     0.0000)


 Task  times  cpu:        0.9s     wall:        1.4s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls: 1173     1173     2748     2320      357        0        0        0     
number of processes/call 1.66e+00 1.39e+00 1.30e+00 0.00e+00 0.00e+00
bytes total:             1.27e+06 1.02e+06 2.31e+05 0.00e+00 0.00e+00 0.00e+00
bytes remote:            4.45e+05 3.38e+05 1.61e+04 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 57680 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        19	        55
	current total bytes		         0	         0
	maximum total bytes		    302416	  22510112
	maximum total K-bytes		       303	     22511
	maximum total M-bytes		         1	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
    J. Hammond, J. Autschbach, F. Aquino, S. Hirata, M. T. Hackler, K. Lopata,
      J. Mullin, P. Nichols, R. Peverati, Y. Zhao, P.-D. Fan, R. J. Harrison,
        M. Dupuis, D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan,
       B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
      G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. Kendall, J. A. Nichols,
        K. Tsemekhman, K. Wolinski, J. Anchell, D. Bernholdt, P. Borowski,
  T. Clark, D. Clerc, H. Dachsel, M. Deegan, K. Dyall, D. Elwood, E. Glendening,
         M. Gutowski, A. Hess, J. Jaffe, B. Johnson, J. Ju, R. Kobayashi,
     R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu,
      L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone, M. Stave, H. Taylor,
                  G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        0.9s     wall:        2.5s
